test/scripts/e2e-kind.sh

```diff
@@ -64,6 +64,10 @@ case "$SHARD" in
 	shard-other)
 		GINKGO_ARGS="${GINKGO_ARGS} "'--ginkgo.focus=\[sig-network\]\s[^NnSs].*'
 		;;
+	shard-test)
```

Comment:
drop "`shard-`" ?

Reply:
That would mean changing the Makefile too. Which seems a bit excessive for only a naming convention